prisma/schema.prisma

```diff
@@ -13,11 +13,25 @@ datasource db {
 
 model Setting {
   id                String      @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
-  bannerImage       String?
+  mediaId           String?     @db.Uuid @unique
```

Comment:
one last request.. Can this be called as bannerImageId and relation be called bannerImage? if we have to add other media in this table, names will be confusing..

Reply:
done